cmd/helpers.go

```diff
@@ -46,3 +52,35 @@ func UserConfirmYes(ctx *cmd.Context) error {
 	}
 	return nil
 }
+
+// UserConfirmName returns an error if we do not read a "name" of the model/controller/etc from user
+// input.
+func UserConfirmName(verificationName string, objectType string, ctx *cmd.Context) error {
+	fmt.Fprintf(ctx.Stdout, nameVerificationMsg, objectType)
+	scanner := bufio.NewScanner(ctx.Stdin)
+	scanner.Scan()
+	err := scanner.Err()
+	if err != nil && err != io.EOF {
+		return errors.Trace(err)
+	}
+	answer := strings.ToLower(scanner.Text())
+	if answer != verificationName {
+		return errors.Trace(userAbortedError("aborted"))
+	}
+	return nil
+}
+
+// CheckSkipConfirmEnvVar check if JujuSkipConfirmationEnvKey environment variable is defined, and if yes -
+// returns its boolean value, if bo - returns false value
+func CheckSkipConfirmEnvVar() (bool, error) {
+	envSkipConfirmValueStr, envVarIsSet := os.LookupEnv(osenv.JujuSkipConfirmationEnvKey)
+	if envVarIsSet {
+		envSkipConfirmValue, err := strconv.ParseBool(envSkipConfirmValueStr)
+		if err != nil {
+			return false, errors.Errorf("Unexpected value of JUJU_SKIP_CONFIRMATION env var, needs to be bool.")
+		}
+		return envSkipConfirmValue, nil
+	} else {
+		return false, errors.NewNotFound(nil, "JUJU_SKIP+CONFIRMATION is not defined.")
```

Comment:
You mean JUJU_SKIP_CONFIRMATION
Also, you should template in the const `osenv.JujuSkip...`

Reply:
Same on line 80

Reply:
Thx, fixed